core/kernel/tee_ta_manager.c

```diff
@@ -112,15 +112,13 @@ static struct tee_ta_ctx *ts_to_ta_ctx(struct ts_ctx *ctx)
 	panic("bad context");
 }
 
-static bool tee_ta_try_set_busy(struct tee_ta_ctx *ctx)
```

Comment:
The func name `tee_ta_try_set_busy` be changed to `tee_ta_try_set_busy_nolock`. Some caller need modify.eg: `tee_ta_set_busy`